frontend/src/hooks/lookup.hooks.js

```diff
@@ -91,6 +91,24 @@ export const useGetNameForCanId = (id) => {
     return displayName;
 };
 
+/**
+ * This hook returns the display name of a Services Component given the id.
+ * @param {number} id - The id of the Services Component.
+ * @returns {string} - The display name of the Services Component.
+ */
+export const useGetNameForServicesComponentId = (id) => {
```

Comment:
🪝we already have this hook in this module: `src/hooks/useServicesComponents.hooks.js`

Reply:
k, I'll get rid of mine and use that one

Reply:
done